mne_connectivity/tests/test_spectral.py

```diff
@@ -114,26 +125,40 @@ def test_spectral_connectivity(method, mode):
                                (fstart - trans_bandwidth * 2,
                                 fend + trans_bandwidth * 2))
         if method == 'coh':
-            assert np.all(con[1, 0, gidx[0]:gidx[1]] > upper_t), \
-                con[1, 0, gidx[0]:gidx[1]].min()
+            assert np.all(
+                con.get_data(output='full')[
+                    1, 0, gidx[0]:gidx[1]
+                ] > upper_t), \
+                con.get_data()[
+                    1, 0, gidx[0]:gidx[1]].min()
             # we see something for zero-lag
-            assert (np.all(con[1, 0, :bidx[0]] < lower_t))
-            assert np.all(con[1, 0, bidx[1]:] < lower_t), \
-                con[1, 0, bidx[1:]].max()
+            assert (np.all(con.get_data(output='full')
+                    [1, 0, :bidx[0]] < lower_t))
+            assert np.all(
+                con.get_data(output='full')[1, 0, bidx[1]:] < lower_t), \
+                con.get_data()[1, 0, bidx[1:]].max()
         elif method == 'cohy':
             # imaginary coh will be zero
-            check = np.imag(con[1, 0, gidx[0]:gidx[1]])
+            check = np.imag(con.get_data(output='full')[1, 0, gidx[0]:gidx[1]])
             assert np.all(check < lower_t), check.max()
             # we see something for zero-lag
-            assert np.all(np.abs(con[1, 0, gidx[0]:gidx[1]]) > upper_t)
-            assert np.all(np.abs(con[1, 0, :bidx[0]]) < lower_t)
-            assert np.all(np.abs(con[1, 0, bidx[1]:]) < lower_t)
+            assert np.all(
+                np.abs(con.get_data(output='full')[
+                    1, 0, gidx[0]:gidx[1]
+                ]) > upper_t)
+            assert np.all(np.abs(con.get_data(output='full')
+                          [1, 0, :bidx[0]]) < lower_t)
+            assert np.all(np.abs(con.get_data(output='full')
+                          [1, 0, bidx[1]:]) < lower_t)
         elif method == 'imcoh':
             # imaginary coh will be zero
-            assert np.all(con[1, 0, gidx[0]:gidx[1]] < lower_t)
-            assert np.all(con[1, 0, :bidx[0]] < lower_t)
-            assert np.all(con[1, 0, bidx[1]:] < lower_t), \
-                con[1, 0, bidx[1]:].max()
+            assert np.all(con.get_data(output='full')[
+                          1, 0, gidx[0]:gidx[1]] < lower_t)
+            assert np.all(con.get_data(output='full')
+                          [1, 0, :bidx[0]] < lower_t)
+            assert np.all(
+                con.get_data(output='full')[1, 0, bidx[1]:] < lower_t), \
```

Comment:
use assert_array_less (changing the code anyway, might as well update to newer best practices)